src/pages/Sign/sections/SignUp.tsx

```diff
@@ -143,7 +143,7 @@ const inputBorderCheck = (
     ? errors
       ? '3px solid red'
       : [`3px solid ${colors.GREEN}`]
-    : [`3px solid ${colors.DARK_PURPLE}`]
+    : [`3px solid ${colors.PURPLE}`]
```

Comment:
```suggestion
    : [`3px solid ${colors.DARK_PURPLE}`]
```

Reply:
As it is handled differently now with focus etc 😊

Reply:
amaze, lets merge that before then!